Samples/office-add-in-commands/auto-open-task-pane/README.md

```diff
@@ -22,7 +22,7 @@ description: "Configure a document to automatically open your Office Add-in's ta
 
```

Comment:
be sure to update the products metadata at the top of this file

Reply:
@ElizabethSamuel-MSFT What needs to change? It looks all still correct.